src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -111,6 +112,20 @@ trait SharedFileSystem {
 
   def ioInterface(workflowOptions: WorkflowOptions): IOInterface = new SharedFileSystemIOInterface
```

Comment:
Not sure if it's worth anything, but I see you call this a few line below when there is a cache hit to get a ioInterface, in this file system the ioInterface doesn't depend on the workflow options (for now at least), so we might want to make a val out of it instead of re-instantiating a new one every time.